misc/docker-ci/check.mk

```diff
@@ -20,9 +27,11 @@ ossl1.1.1:
 		CMAKE_ARGS='-DOPENSSL_ROOT_DIR=/opt/openssl-1.1.1'
 
 dtrace:
+	rm -rf src/h2olog/generated_raw_tracer.cc # make sure it's up-todate
```

Comment:
I'm sorry! I forgot to push some of my changes. 

As you said,  there are cases where we fail to generate `generated_raw_tracer.cc`, and I've fixed `CMakeLists.txt` instead of removing the file. 210e1136c1e4e3606d79cdf554a203bd7a8d5ba0

Reply:
Once again, a separate PR?

Reply:
Well, agreed!